coalib/results/TextPosition.py

```diff
@@ -31,3 +31,48 @@ def line(self):
     @property
     def column(self):
         return self._column
+
+    def __le__(self, other):
+        """
+        Test whether self is behind the other TextPosition.
```

Comment:
`self` could be also enclosed in double backticks :) (counts for all occurrences)